AWS/legos/aws_delete_ebs_snapshot/aws_delete_ebs_snapshot.json

```diff
@@ -0,0 +1,12 @@
+{
+    "action_title": "AWS Delete EBS Snapshot",
+    "action_description": "Delete EBS Snapshot for an EC2 instance",
+    "action_type": "LEGO_TYPE_AWS",
+    "action_entry_function": "aws_delete_ebs_snapshot",
+    "action_needs_credential": true,
+    "action_output_type": "ACTION_OUTPUT_TYPE_LIST",
+    "action_is_remediation": true,
```

Comment:
what is is_remediation?

Reply:
error. removed it.

Reply:
So for all the remediation legos, we are adding `action_is_remediation` .